src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -1098,5 +1098,138 @@
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/locations/directions.asp",
+    "dest": "/altoona-health-care/locations/james-e-van-zandt-veterans-administration-medical-center/"
+  },
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/locations/DuBois.asp",
+    "dest": "/altoona-health-care/locations/dubois-va-clinic/"
+  },
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/locations/Huntingdon_VA_Community_Based_Outpatient_Clinic.asp",
+    "dest": "/altoona-health-care/locations/huntingdon-county-va-clinic/"
+  },
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/locations/Indiana_VA_Outpatient_Clinic.asp",
+    "dest": "/altoona-health-care/locations/indiana-county-va-clinic/"
+  },
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/locations/Johnstown.asp",
+    "dest": "/altoona-health-care/locations/johnstown-va-clinic/"
+  },
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/locations/StateCollege.asp",
+    "dest": "/altoona-health-care/locations/state-college-va-clinic/"
+  },
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/services/covid-19-vaccines.asp",
+    "dest": "/altoona-health-care/programs/covid-19-vaccines/"
+  },
+  {
+    "domain": "www.wilmington.va.gov",
+    "src": "/",
+    "dest": "/wilmington-health-care/",
+    "catchAll": true
+  },
+  {
+    "domain": "www.wilmington.va.gov",
+    "src": "/locations/directions.asp",
+    "dest": "/wilmington-health-care/locations/wilmington-va-medical-center/"
+  },
+  {
+    "domain": "www.wilmington.va.gov",
+    "src": "/locations/Atlantic_County_Community_Based_Outpatient_Clinic.asp",
+    "dest": "/wilmington-health-care/locations/atlantic-county-va-clinic/"
+  },
+  {
+    "domain": "www.wilmington.va.gov",
+    "src": "/locations/Cape_May_County_Community_Based_Outpatient_Clinic.asp",
+    "dest": "/wilmington-health-care/locations/cape-may-county-va-clinic/"
+  },
+  {
+    "domain": "www.wilmington.va.gov",
+    "src": "/locations/Cumberland_County_Community_Based_Outpatient_Clinic.asp",
+    "dest": "/wilmington-health-care/locations/cumberland-county-va-clinic/"
+  },
+  {
+    "domain": "www.wilmington.va.gov",
+    "src": "/locations/Kent_County_Community_Based_Outpatient_Clinic.asp",
+    "dest": "/wilmington-health-care/locations/kent-county-va-clinic/"
+  },
+  {
+    "domain": "www.wilmington.va.gov",
+    "src": "/locations/Sussex_County_Community_Based_Outpatient_Clinic.asp",
+    "dest": "/wilmington-health-care/locations/sussex-county-va-clinic/"
+  },
+  {
+    "domain": "www.wilmington.va.gov",
+    "src": "/services/covid-19-vaccines.asp",
+    "dest": "/wilmington-health-care/programs/covid-19-vaccines/"
+  },
+  {
+    "domain": "www.wilkes-barre.va.gov",
+    "src": "/",
+    "dest": "/wilkes-barre-health-care/",
+    "catchAll": true
+  },
+  {
+    "domain": "www.wilkes-barre.va.gov",
+    "src": "/locations/directions.asp",
+    "dest": "/wilkes-barre-health-care/locations/wilkes-barre-va-medical-center/"
```

Comment:
Let's remove this 1:1 redirect, we do not need to redirect the legacy directions page.